pkg/lhutil/lighthouse.go

```diff
@@ -20,6 +20,7 @@ package lhutil
 
 import (
 	"fmt"
+	corev1 "k8s.io/api/core/v1"
 	mcsv1a1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
 
```

Comment:
probably bad left over from my commits but should run go imports to fix empty line and order

Reply:
yes, i will make linter happy now and push again.